package.json

```diff
@@ -14,6 +14,7 @@
     "publish": "npm run clean:build && lerna publish -m 'chore: Publish [skip ci]' --no-git-reset --conventional-commits --yes"
   },
   "dependencies": {
+    "@azure/arm-communication": "^3.0.1",
```

Comment:
We don't want to add arm-communication to the top level package. This should only be in your ingredient.